lib/webauthn/attestation_statement/packed.rb

```diff
@@ -86,6 +87,16 @@ def valid_signature?(authenticator_data, client_data_hash)
       def verification_data(authenticator_data, client_data_hash)
         authenticator_data.data + client_data_hash
       end
+
+      def attestation_type_and_trust_path
+        if raw_attestation_certificates&.any?
+          [ATTESTATION_TYPE_BASIC_OR_ATTCA, attestation_certificate_chain]
+        elsif raw_ecdaa_key_id
+          [ATTESTATION_TYPE_ECDAA, raw_ecdaa_key_id]
```

Comment:
We don't support `ECDAA` for now.
Can we please remove it from here, it will never execute this code.